apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -47,17 +47,121 @@ const validators = [
   },
 ];
 
-for (const account of e2eDelegationAccounts) {
-  test.describe("Delegate", () => {
+test.describe("Delegate flows", () => {
+  test.beforeAll(async () => {
+    if (new Date().getDay() === 1) {
```

Comment:
but this means that any execution that happens on day 1 will have ENABLE_TRANSACTION_BROADCAST = true, and we don't want this, we want only broadcast once during nightly and that's it, also for release validation we need to have ability to include/enable broadcasting no matter the day

Reply:
indeed, added an additional check
will test tomorrow